.github/workflows/dart_package.yml

```diff
@@ -77,7 +77,7 @@ jobs:
       - name: 🧪 Run Tests
         run: |
           dart pub global activate coverage 1.2.0
-          dart test -j ${{inputs.concurrency}} --coverage=coverage --platform=${{inputs.platform}} && dart pub global run coverage:format_coverage --lcov --in=coverage --out=coverage/lcov.info --packages=.dart_tool/package_config.json --report-on=${{inputs.report_on}}
+          dart test -j ${{inputs.concurrency}} --coverage=coverage --platform=${{inputs.platform}} && dart pub global run coverage:format_coverage --lcov --check-ignore --in=coverage --out=coverage/lcov.info --packages=.dart_tool/package_config.json --report-on=${{inputs.report_on}}
```

Comment:
We should add this under an option as this could be considered breaking for anyone using this in the next version

Reply:
OK, I added a new input for this.

Reply:
@wolfenrain I think we should go forward with the breaking change. 

I think it is more sensible to have it enabled by default. Since if you are using comments to ignore particular sections of the code you're most likely doing it purposely already.